_data/conference.yml

```
@@ -211,34 +212,61 @@ subscribe-title: Join our conference
+    description: While in the city, explore more of Lausanne! Lausanne is home to the Olympic Museum, offers stunning views of Lake Geneva, and is known for its vibrant cultural scene and beautiful parks, making it one of Switzerland's most picturesque cities.
 
 the-venue-hero-description: The conference is organised by the Scala Center at the SwissTech Convention Center, in the heart of campus at EPFL (École Polytechnique Fédérale de Lausanne).
 the-venue-hero-cta: Learn more about the Venue
 
 the-venue-title: An awesome venue
-the-venue-subtitle: MEUU
-the-venue-description: MEEU is located at the Chamartín train station with state-of-the-art infrastructure and technology at the center of the Spanish capital's financial district. Its convenient location allows fast transportation to easily get anywhere in Madrid or the rest of Spain. The MEEU space is versatile and functional, with large outdoor areas. Along with its convenient location, MEEU is surrounded by great restaurants and hotels. You can find MEEU at Calle de Agustín de Foxá, 40, 28036 Madrid.
+the-venue-subtitle: Swisstech Convention Center
+the-venue-description: The SwissTech Convention Center (STCC) is located on the EPFL campus, featuring state-of-the-art infrastructure and technology at the heart of the Lake Geneva region's innovation hub. Its strategic location provides excellent connectivity, with easy access to Lausanne’s city center and nearby cities like Geneva. The STCC offers a versatile and functional space with modern facilities designed to accommodate a wide range of events. Along with its convenient location, the area surrounding the STCC features various restaurants, hotels, and leisure options. You can find the STCC at Route Louis-Favre 2, 1024 Ecublens.
 the-venue-cta: Visit the website
-the-venue-url: https://www.meeu.es/
-the-venue-image: img/assets/madrid/the-venue-madrid.jpg
+the-venue-url: https://www.stcc.ch/
+the-venue-image: img/assets/the-venue-lausanne.jpg
 
 
 # Sponsor
 # ----------------------------------------------------------------------------------------------------------
 # ----------------------------------------------------------------------------------------------------------
 sponsor-hero-description: Thanks for your interest in sponsorship opportunities for Scala Days.
 sponsor-hero-cta: Sponsorship Packages
+
+# Tickets
+# ----------------------------------------------------------------------------------------------------------
+# ----------------------------------------------------------------------------------------------------------
+tickets:
+  - icon: img/assets/icon-airport.svg
+    title: Visa/Invitation Letter
+    description: An invitation letter for your visa application procedure is available after completing the registration process.
```

This is confusing to me, I recall people should reach out to Lausanne Tourisme after registration.
Maybe consider changing the sentence to say:
```suggestion
    description: An invitation letter for your visa application procedure can be requested after completing the registration process.
```

---

The text is a copy/paste of what was provided to me by @allyalexandrou